Misc/NEWS

```diff
@@ -306,6 +306,10 @@ Extension Modules
 
 Library
 -------
+
+- bpo-29694: Fixed race condition in pathlib mkdir with flags
+  parents=True.
```

Comment:
Since Armin is a core developer and will merge the PR himself this is not necessary.